### ISSUE TITLE
Naive question

### ISSUE THREAD

**Issue by dlongm01** [files-community/files-community.github.io#61]
Hi

Apologies is this is the wrong location for this question or if it is too naive, but I have installed Files (which looks very good) and set it to be my default app using the regedit script.

However, my programs such as MS Office or others such as Calibre seem to continue to use the 'old style' Windows File Explorer when saving/opening files.

I'm guessing this is set up when programs are first installed but is there a way to change the default files program for existing software (or 'apps')?

Or are my expectations wrong?

Thanks

**Comment by chingucoding** [comment]
Hi,

Apps such as Word or Calibre often use a control called "File Picker" for operations such as saving or opening files. That "File Picker" control is part of the UI framework being used, so these apps don't actually request the Windows File Explorer but rather talk to their UI framework which then spawns this flyout. As far as I know, this is not something that can be customized using regex keys since this is something backed into the Windows UI frameworks. I'm afraid there isn't much files can do here.

But please take my comment with a grain of salt, I am not 100% sure that there is no way, I haven't found one yet though. @yaichenbaum @duke7553 please feel free to chime in and correct me if you know if there is something that could be done!

**Comment by chingucoding** [comment]
By the way, dlongm01, you are not the first that wishes to switch the default file picker used by something more modern! See this discussion (https://github.com/microsoft/microsoft-ui-xaml/issues/2854) which talks about replacing the default file picker being used in UWP/WinUI apps.

**Comment by dlongm01** [comment]
Hi @chingucoding - thanks for the quick reply. That makes sense (to my limited understanding) and I'll just keep an eye on things. Following up your second post it looks as though some folk are thinking about this, so something might happen.

I guess I was  bit anxious that I hadn't done the default app setting properly as W10 gives no feedback and it takes a slightly confusing approach to the process. The Regedit script was helpful though.

I'll suck it and see and I'm still learning about the features that Files provides ....

Cheers

**Comment by chingucoding** [comment]
Yeah, the process is very confusing, but it also seems that Windows isn't too keen right now on having that be customizable (after all, one of the easier ways is editing the registry...). Hopefully this changes sooner or later making it easier to have Files the default experience for this,

**Comment by yaichenbaum** [comment]
@dlongm01 @chingucoding Thanks for bringing up this discussion, while we haven't implemented any support, there are ways to hook into the shell and redirect the file picker to use Files instead. We don't have plans to implement support, but that doesn't mean someone else won't 🙂.
